packages/wonder-blocks-form/components/choice-internal.js

```diff
@@ -146,16 +148,23 @@ export default class ChoiceInternal extends React.Component<Props> {
 const styles = StyleSheet.create({
     wrapper: {
         flexDirection: "row",
-        alignItems: "center",
+        alignItems: "flex-start",
         outline: "none",
     },
+    label: {
+        // NOTE: The checkbox/radio button (height 16px) should be center
+        // aligned with the first line of the label. However, LabelMedium has a
+        // declared line height of 20px, so we need to adjust the top to get the
+        // desired alignment.
+        marginTop: -2,
```

Comment:
🤷‍♂️ Hmm - this feels a bit awkward, is there any way we could use flexbox to achieve this?

Reply:
Not sure -- https://stackoverflow.com/questions/32118013/align-icon-vertically-to-the-center-of-the-first-line-of-text suggests changing the `line-height` of the label, but that's probably not what we want to affect.

Reply:
Unfortunately it's hard to actually calculate how CSS line-height affects the positioning of the text relative to other things on the line.  See https://iamvdo.me/en/blog/css-font-metrics-line-height-and-vertical-align for the gory details.